### ISSUE TITLE
Format on Save

### ISSUE THREAD

**Issue by amosyuen** [mrmlnc/vscode-stylefmt#116]
Great plugin, could you add an option to format on save?

**Comment by palashkaria** [comment]
@amosyuen If you are looking for a quick solution, install [run on save plugin](https://marketplace.visualstudio.com/items?itemName=emeraldwalk.RunOnSave) and add something like this you your user settings:

```
"emeraldwalk.runonsave": {
        "commands": [{
                "match": "\\.scss?$",
                "cmd": "cd ${workspaceRoot} && stylefmt -c .stylelintrc ${file}"
        }]
}
```

**Comment by maliMirkec** [comment]
@palashkaria thank you for a great suggestion.
Just make sure to install Stylefmt locally or globally. Otherwise, this command won't work.

**Comment by mrmlnc** [comment]
Fixed by #162.
